pkg/tag/tag_test.go

```diff
@@ -56,7 +56,7 @@ func TestGetTagAsFilter(t *testing.T) {
 	actualTagFilter, err := tag.GetTagAsFilter(*tags)
 	now := time.Now()
 	zone, _ := now.Zone()
-	currTimeStr := fmt.Sprintf("%d-%d-%d %d:%d:%d %s", now.Year(), now.Month(), now.Day(), now.Hour(), now.Minute(),
+	currTimeStr := fmt.Sprintf("%04d-%02d-%02d %02d:%02d:%02d %s", now.Year(), now.Month(), now.Day(), now.Hour(), now.Minute(),
```

Comment:
@GavinBurris42 This one I copied over directly from your fork https://github.com/GavinBurris42/aws-simple-ec2-cli/pull/1/files so we can get it into main faster.